### PR TITLE
binlog replication documentation

### PR DESCRIPTION
**Pull request by fulghum**
Added a new guide for binlog replication and spruced up a few other replication related pages.

### CODE REVIEW

File: content/SUMMARY.md [hunk 1]
```diff
@@ -71,6 +71,7 @@
   - [Install DoltLab](guides/doltlab/installation.md)
   - [Administrator Guide](guides/doltlab/administrator.md)
 - [Importing Data](guides/import.md)
+- [MySQL Replication](guides/binlog-replication.md)
```
Comment (timsehn) [inline content/SUMMARY.md]:
```suggestion
- [MySQL to Dolt Replication](guides/binlog-replication.md)
```

**Review by timsehn — APPROVED** [review]
Just a title suggestion. Otherwise, ship it.

File: content/reference/sql/server/replication.md [hunk 2]
```diff
@@ -4,32 +4,45 @@ title: Replication
 
 # Replication
 
-Dolt supports two forms of
-[replication](../../../concepts/dolt/rdbms/replication.md).
+Dolt supports two forms of [replication](../../../concepts/dolt/rdbms/replication.md) where a Dolt primary sql-server replicates changes to Dolt sql-server replicas. If you're looking to replicate changes from a MySQL primary server to a Dolt replica, check out the guide to [MySQL Replication to Dolt](../../../guides/binlog-replication.md).
```
Comment (zachmu) [inline content/reference/sql/server/replication.md]: Think I would introduce this as, "Dolt can replicate between 2 or more Dolt servers, or can be a read-replica for a MySQL server."

File: content/reference/sql/server/replication.md [hunk 3]
```diff
@@ -4,32 +4,45 @@ title: Replication
...
+In **Direct-to-Standby Replication**, the primary dolt sql-server instance replicates all writes to a
 set of configured standby servers. In this mode, there is no intermediate
 [remote](../../../concepts/dolt/git/remotes.md) and all SQL transaction commits
 are replicated, not just Dolt commits.
 
 ![Standby replication](../../../.gitbook/assets/dolt-standby-replication.png)
 
-This page describes configuration and considerations for both types of
+Use this form of replication when you have high-availability requirements, need a hot standby
+server ready to swap in for the primary, and you do **not** need to replicate from a MySQL primary
```
Comment (zachmu) [inline content/reference/sql/server/replication.md]: I wouldn't mention MySQL at all here, make its own section

File: content/reference/sql/server/replication.md (hunk 2 above)
Comment (zachmu) [reply content/reference/sql/server/replication.md]: I would add a brief "Replicating from a MySQL server" section at the end of this doc, mostly just a link to the guide

File: content/guides/binlog-replication.md [hunk 4]
```diff
@@ -0,0 +1,98 @@
...
+
+![Binlog replication](../.gitbook/assets/dolt-binlog-replication.png)
+
+
+## Requirements for MySQL Primary Server
+
+**Row-Based Replication** – MySQL provides two main formats for binlog event recording (and a third, “mixed” format that combines the two). Dolt's binlog replication requires that binlog events are configured for row-based replication (the default in MySQL 8.0+). Other binlog formats (e.g. statement based replication) will not work correctly.
+**GTID-Based Auto-positioning** – MySQL provides two ways for coordinating the replication position in a binlog event stream. The older method relies on specifying the binlog file name and file position; the newer, recommended version relies on Global Transaction Identifiers (GTIDs). Auto-positioning allows the replica to find its position from the binlog event stream, without requiring additional configuration.
```
Comment (zachmu) [inline content/guides/binlog-replication.md:14]: Need a blank line between sections for them to be new paragraphs

File: content/guides/binlog-replication.md [hunk 5]
```diff
@@ -0,0 +1,98 @@
...
+
+View replication status (`\G` is useful here for printing results vertically): `SHOW REPLICA STATUS \G`
+
+Stop replication: `STOP REPLICA;`
+
+Clear out replication source and filtering configuration: `RESET REPLICA ALL;`
+
+
+## Limitations
+**Replica mode only** – Dolt can only consume binlog events as a replica; it cannot act as a primary and produce binlog events.
```
Comment (zachmu) [inline content/guides/binlog-replication.md:78]: Same blank line formatting issue here

**Review by zachmu — APPROVED** [review]
Great docs

Just some minor comments

File: content/reference/sql/server/replication.md (hunk 3 above)
Comment (fulghum) [reply content/reference/sql/server/replication.md]: Whoops! Good catch. I meant to take that part out after I moved the binlog content out and into a guide. Will fix.

**Pull request merged**